x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsLicenseIntegTests.java

```diff
@@ -89,6 +89,9 @@ public void createAndMountSearchableSnapshot() throws Exception {
 
         assertAcked(client().execute(DeleteLicenseAction.INSTANCE, new DeleteLicenseRequest()).get());
         assertAcked(client().execute(PostStartBasicAction.INSTANCE, new PostStartBasicRequest()).get());
+
+        ensureClusterSizeConsistency();
```

Comment:
NIT: I think we can do it in one line with `ensureStableCluster(cluster().size())`? That seems like it's all we need here? Looks like this is a retry on the master node request I guess? (though there's no master fail-over, but maybe a connection got dropped for some reason?) ... That's my best guess :)

Reply:
I merged as is, I wanted to ensure no pending tasks on all nodes with the same cluster state everywhere. 

And yes, retries might explain this. We'll see if that happens again